src/database/entity/gamebanana_maps.js

```diff
@@ -0,0 +1,30 @@
+var EntitySchema = require('typeorm').EntitySchema;
+
+module.exports = new EntitySchema({
+    name: 'gamebanana_maps', // Will use table name `category` as default behaviour.
+    columns: {
+        id: {
+            primary: true,
+            type: 'int',
+            generated: true,
+        },
+        modID: {
+            type: 'int',
+        },
+        mapName: {
+            type: 'varchar',
+        },
+        mapAuthor: {
+            type: 'varchar',
+        },
+        mapPreviewImage: {
+            type: 'varchar',
+        },
```

Comment:
Storing image URL in the database is not really needed as we can recreate that using the mod ID.